### PR TITLE
[blog] New post for June

### PR DESCRIPTION
**Pull request by beets**
A preview:

image

### CODE REVIEW

File: _posts/2021-06-01-updates.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+author: Carolyn Au
+title: Data Commons Updates
+---
+
+We've been hard at work since we [surfaced Data Commons in Google Search](https://docs.datacommons.org/2020/10/15/search_launch.html) last October. Some of the exciting features we've added include:
+
+
+### Internationalization support
+
+Place Explorer is now available in 8 languages in addition to English: [German](https://datacommons.org/place/country/DEU?hl=de), [Spanish](https://datacommons.org/place/country/MEX?hl=es), [French](https://datacommons.org/place/country/FRA?hl=fr), [Hindi](https://datacommons.org/place/country/IND?hl=hi), [Italian](https://datacommons.org/place/country/ITA?hl=it), [Japanese](https://datacommons.org/place/country/JPN?hl=ja), [Korean](https://datacommons.org/place/country/KOR?hl=ko) and [Russian](https://datacommons.org/place/country/RUS?hl=ru). Additionally, support for these languages are carried forward from searches on Google Search, [here's an example](https://www.google.com/search?hl=es&q=Poblaci%C3%B3n+de+M%C3%A9xico).
```
Comment (shifucun) [inline _posts/2021-06-01-updates.md:12]: "from searches on Google Search" could this just be "from Google Search"?

File: _posts/2021-06-01-updates.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+author: Carolyn Au
+title: Data Commons Updates
+---
+
+We've been hard at work since we [surfaced Data Commons in Google Search](https://docs.datacommons.org/2020/10/15/search_launch.html) last October. Some of the exciting features we've added include:
+
+
+### Internationalization support
```
Comment (shifucun) [inline _posts/2021-06-01-updates.md:10]: support -> Support?

File: _posts/2021-06-01-updates.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+### New Scatter Plot Explorer
+
+The new [Scatter Plot Explorer](https://datacommons.org/tools/scatter) enables quick visual exploration of any two statistical variables for a set of places. Try it out for [Bachelor Degree Attainment vs Females per capita in California Counties](https://datacommons.org/tools/scatter#%26svx%3DCount_Person_Female%26svpx%3D0-8-0%26svdx%3DCount_Person%26svnx%3DFemale%26pcx%3D1%26svy%3DCount_Person_EducationalAttainmentBachelorsDegree%26svpy%3D2-0-6%26svdy%3DCount_Person_25OrMoreYears%26svny%3DBachelors_Degree%26pcy%3D1%26epd%3DgeoId%2F06%26epn%3DCalifornia%26ept%3DCounty) or [Covid-19 cases vs African Americans Per Capita among US States](https://datacommons.org/tools/scatter#%26svx%3DCount_Person_BlackOrAfricanAmericanAlone%26svpx%3D0-14-2%26svdx%3DCount_Person%26svnx%3DBlack_Or_African_American_Alone%26pcx%3D1%26svy%3DCumulativeCount_MedicalTest_ConditionCOVID_19_Positive%26svpy%3D5-2-0-1%26svdy%3DCount_Person%26svny%3DPositive%26pcy%3D1%26epd%3Dcountry%2FUSA%26epn%3DUnited%20States%20of%20America%26ept%3DState).
```
Comment (shifucun) [inline _posts/2021-06-01-updates.md:23]: "per capita" lower case for consistency?

File: _posts/2021-06-01-updates.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+We have also released a new set of APIs centered around statistics retrieval. There are different REST endpoints to retrieve [a single statistical value](https://docs.datacommons.org/api/rest/stat_value.html), [a statistical time series](https://docs.datacommons.org/api/rest/stat_series.html) or [the entire collection of statistical data for a set of places](https://docs.datacommons.org/api/rest/stat_all.html). We have used these APIs to build the new Scatter Plot Explorer and hope this enables other applications too.
+
+
+
+### New Data
+
+As always, we continue to add more data to the Data Commons Graph. Some recent additions include:
+- Indian Census (e.g. [houseless](https://datacommons.org/browser/country/IND?statVar=Count_Household_Houseless_Rural) and [rural literacy](https://datacommons.org/browser/wikidataId/Q15116?statVar=Count_Person_Literate_Rural) populations)
+- Reserve Bank of India's Poverty data (e.g. [Rural population below poverty in Andhra Pradesh](https://datacommons.org/browser/wikidataId/Q1159?statVar=Count_Person_Rural_BelowPovertyLevelInThePast12Months))
```
Comment (shifucun) [inline _posts/2021-06-01-updates.md:43]: Rural -> rural

File: _posts/2021-06-01-updates.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - BLS Unemployment (till [2021 Q1](https://datacommons.org/browser/geoId/06?statVar=UnemploymentRate_Person))
+  - World Bank (till [2019](https://www.google.com/search?q=population+of+china&oq=population+of+china))
+
+
+
+As always, we are eager to hear from you! Please [share your feedback](https://datacommons.org/feedback) with our team.
+
+
+
+Carolyn on behalf of the Data Commons team
```
Comment (shifucun) [inline _posts/2021-06-01-updates.md:59]: nit: add period

**Review by shifucun — COMMENTED** [review]
Nice update! Just some nit suggestions.

**Review by chejennifer — APPROVED** [review]
nice!

File: _posts/2021-06-01-updates.md (hunk 5 above)
Comment (beets) [reply _posts/2021-06-01-updates.md:59]: skipping this since it's the signature line and not really a sentence

**Review by beets — COMMENTED** [review]
thanks for catching those nits!

**Pull request merged**